_dashboards/visualize/viz-index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Visualize
+nav_order: 4
+has_children: true
+---
+
+# Visualize
+
+Data visualization is representing your data in a visual context. By visualizing your data, you translate complex, high-volume, or numerical data into a visual representation that is easier to process.
+
+OpenSearch Dashboards gives you data visualization tools to improve and automate the visual communication process. Using visual elements like charts, graphs, or maps to represent data gives you the tools to advance business intelligence and support data-driven decision-making and strategic planning.
+
+# Visualization types in OpenSearch Dashboards
```

Comment:
It might be worth it to separate the built in visualization types i.e. Bar, Line etc from the ones that come from plugins e.g. Gantt. Maps will soon be (or is already) a part of this list.

Reply:
@ashwin-pc Which types are built-in and which types are plugins? Where do I find this information?

Reply:
There's no good resource on that without reading the code, unfortunately. The only one on this list is `Gantt charts`.

Reply:
I'm working on subpages for each type or tool. Are you all okay with keeping this page higher level and then addressing these distinctions in the subpage content?